src/library/export/trackexportworker.cpp

```diff
@@ -1,17 +1,16 @@
 #include "library/export/trackexportworker.h"
 
+#include <QDebug>
```

Comment:
Please avoid to reformat existing files that haven't been modified. This will produce many unrelated changes that don't need to be review.

Reply:
Thanks - I think this was over-eagerness with clang-format.  I will revert.

Reply:
It's possible that this was done automatically by pregcommit when merging master (because we merged commits by someone else who was not using pre-commit). In that case it's okay IMO.

Reply:
The commit that did this (`daa23671`) is in my branch history, by another contributor (whose changes have mostly been reverted), so I can wind this back.

In fact, I will try cleaning up the whole history, as very few of the older commits add any value on their own and it would probably benefit from a larger rebase onto master.

Reply:
I've cleaned up the history significantly into a single, meaningful commit.  This resolves the issue with unnecessary reformatting noise on the track-related export files.